src/index.ts

```diff
@@ -41,8 +44,10 @@ const controls: Record<string, () => void> = {
   o: () => cameraController.round(),
 }
 
+const isValidControlKey = (key: string): key is ControlsKeys => key in controls
```

Comment:
I prefer creating this as a regular function instead of an arrow function as it's more readable, what u think?

Reply:
@diego3g updated :)